web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -133,7 +134,11 @@ export class LayerDialogComponent implements OnDestroy {
     const canAddPoints = this.layer?.contributorsCanAdd?.find(
       val => val === 'points'
     );
-    this.contributorsCanAdd = canAddPoints ? true : false;
+    const canAddPolygons = this.layer?.contributorsCanAdd?.find(
+      val => val === 'polygons'
+    );
+    this.contributorsCanAddPoint = canAddPoints ? true : false;
```

Comment:
=> `this.contributorsCanAddPoint = this.layer?.contributorsCanAdd?.includes('points')`

Same for Polygons. You can then delete the above calls to `find`

Reply:
updated